lib/Test2/Util.pm

```diff
@@ -143,6 +145,15 @@ sub pkg_to_file {
 
 sub ipc_separator() { "~" }
 
+BEGIN {
+    if ($Config{sig_name} =~ m/\bsys\b/) {
```

Comment:
At least on Mac and Linux where I've spot checked, sig_name contains uppercase values.  Maybe this should be a case-insensitive match?